master/usage/migrating-pools.md

```diff
@@ -0,0 +1,116 @@
+
+### Purpose of this page
+
+Provide guidance on how to migrate off of a live Calico pool.
+
+## Prerequisites
+
+**Orchestrator Support**
+
+While calico supports changing the live address range, not all orchestrators do. 
+Be sure to consult the documentation for whichever installer you are using to ensure it supports changing it. 
+
+For example, in Kubernetes, you will also need to change:
+- kube-apiserver's `--pod-network-cidr`
+- kube-proxy & kube-controller-manager's `--cluster-cidr`
+
+**Application Availability Impact**
+
+This process wil require the recreation of all Calico-networked pods, which will have some impact on availability
+of your applications.
+
+**Consequences of deleting of an active Pool without following this migration procedure**
+
+Removing a pool actively being used will be bad. If the pool is removed before all workloads are moved, those workloads may experience networking difficulties, namely Calico will no longer NAT the traffic (if nat-outgoing is enabled) or properly encapsulate traffic (if using IP-IP mode).
+
+Removing an active pool without following the migration procedure will cause network connectivity disruptions in running 
+workloads. Namely, Calico will no longer NAT the traffic (if nat-outgoing is enabled) or properly encapsulate traffic (if using IP-IP mode).
+
+## Migrating off an active pool
+
+The basic process is as follows:
+
+1. Add a new pool.
+2. Disable the old pool.
+3. Recreate all existing workloads that were assigned an address from the old pool.
+4. Remove the old pool.
+
+### Example: Kubernetes
+
+In this example, we just provisioned a Kubernetes cluster using Kubeadm, and realized we set `--pod-network-cidr=10.0.0.0/16`, but forgot to set the default Calico pool to match,
+so kube-dns is running on the wrong address range:
+
+```
+# calicoctl get wep --all-namespaces 
+NAMESPACE     WORKLOAD                   NODE      NETWORKS            INTERFACE         
+kube-system   kube-dns-6f4fd4bdf-8q7zp   vagrant   192.168.52.130/32   cali800a63073ed   
+
+# calicoctl get ippool -o wide
+NAME                  CIDR             NAT    IPIPMODE   DISABLED   
+default-ipv4-ippool   192.168.0.0/16   true   Always     false      
+```
+
+Let's get started.
+
+1. Add a new pool:
+
+   ```
+   calicoctl create -f -<<EOF
+   apiVersion: projectcalico.org/v3
+   kind: IPPool
+   metadata:
+     name: new-pool
+   spec:
+     cidr: 10.0.0.0/16
+     ipipMode: Always
+     natOutgoing: true
+     disabled: false
+   EOF
+   ```
+
+   We should now have two enabled pools:
+
+   ```
+   # calicoctl get ippool
+   NAME                  CIDR             
+   default-ipv4-ippool   192.168.0.0/16   
+   new-pool              10.0.0.0/16   
+   ```
+
+2. Disable the old pool. This can be done by appending `disabled: true` to the pool's yaml. 
+
+   ```
+   { calicoctl get ippool default-ipv4-ippool -o yaml; echo "  disabled: true";} | calicoctl apply -f -
```

Comment:
That's a fairly advance shell construction.  I think it'd help the user to understand if you spelled this out: 

    First get the existing pool:

    <dump of existing pool>

    Edit the file to add `disable: true`

    <dump of edit with change highlighted>

    Then apply the update.